main/options/options.cc

```diff
@@ -402,6 +414,10 @@ buildOptions(const vector<pipeline::semantic_extension::SemanticExtensionProvide
     options.add_options("dev")("stop-after", to_string(all_stop_after),
                                cxxopts::value<string>()->default_value("inferencer"), "phase");
     options.add_options("dev")("no-stdlib", "Do not load included rbi files for stdlib");
+    options.add_options("dev")("minimize-to-rbi",
+                               "[experimental] Output a minimal RBI contining the diff between Sorbet's view of a "
+                               "codebase and the definitions present in this file",
```

Comment:
In the future, I think it makes sense to fix the TODOs in `minimize.cc` and mark this not experimental, so that people could "minimize" a sorbet-typed RBI and a reflection generated RBI.

Right now, there are a lot of problems with it, because it aims to be a replacement for the Ruby version in Stripe's codebase, which never needed to support a lot of other features.